modules/release-notes/pages/0.6.24-rn.adoc

```diff
@@ -39,40 +69,48 @@ You can use this option when running the {IC} network locally or connected to th
 +
 For example, you might run the following `+dfx canister create+` command to initialize `+8000000000+` cycles for all of the canisters in a project:
 +
-[source, bash]
-----
+....
 dfx canister create --with-cycles 8000000000 --all
-----
+....
 +
 If using `+dfx deploy+`, you might run the following command to initialize `+8000000000+` cycles for the `+backend+` canister in a project:
 +
-[source, bash]
-----
+....
 dfx deploy --with-cycles 8000000000 backend
-----
+....
 +
 
 NOTE: You must have a cycles wallet canister with a cycles balance on the network where you want to create or deploy additional canisters.
 
 * You can now use the new `+dfx toolchain+` command to manage the version of the `+dfx+` command-line interface you are using for your projects.
 +
-The dfx toolchain command enables you to install, uninstall, and set the default version of `dfx` that you want to use. 
+The `+dfx toolchain+` command enables you to install, uninstall, and set the default version of `dfx` that you want to use. 
 You can specify the version by the complete version number, the major and minor version number, or a tag name.
 For example:
-
++
 ....
 dfx toolchain install 0.6.24 # complete version
 dfx toolchain install 0.6    # major minor version
 dfx toolchain install latest # tag name
 ....
 
+* A new `+deploy-wallet+` subcommand enables you to specify the canister identifier for your cycles wallet WebAssembly module (WASM).
++
+For example, if you have an account with a third party exchange such as Coinbase and receive a wallet canister identifier, you can run a command similar to the following to deploy the wallet and uses its cycles for development:
```

Comment:
Can we specify here that this is a command which is not applicable for the current IC running on sodium but intended for future use. We should omit specific mention of Coinbase as well